From f1c69f00a3dcbcb6f96c8f4f0f5f6c4d54302cd5 Mon Sep 17 00:00:00 2001
From: Marc Mutz <marc.mutz@qt.io>
Date: Wed, 29 Oct 2025 16:36:11 +0100
Subject: [PATCH 543/553] tst_QVariant: fix Clang -Wunused-const-variable

Amends 42e0f9811095fc73c5c1209632247446e6229b3a, which removed the
(last) user of the variable.

Pick-to: 6.8 6.5
Change-Id: If8fa5150cc936ddda2bdccb9d7e400c631831e3a
Reviewed-by: Ahmad Samir <a.samirh78@gmail.com>
(cherry picked from commit 8f8b6e50c844eccbe8557be2b13affb6c7da6275)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 tests/auto/corelib/kernel/qvariant/tst_qvariant.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/auto/corelib/kernel/qvariant/tst_qvariant.cpp b/tests/auto/corelib/kernel/qvariant/tst_qvariant.cpp
index 9afd084cbd0..bb3ad72bb8e 100644
--- a/tests/auto/corelib/kernel/qvariant/tst_qvariant.cpp
+++ b/tests/auto/corelib/kernel/qvariant/tst_qvariant.cpp
@@ -455,7 +455,6 @@ private:
 };
 
 const qlonglong intMax1 = (qlonglong)INT_MAX + 1;
-const qulonglong uintMax1 = (qulonglong)UINT_MAX + 1;
 
 void tst_QVariant::constructor()
 {
-- 
2.51.2

